tests/service/ProjectOptionsTests.fs

```diff
@@ -199,7 +172,9 @@ let ``Project file parsing -- reference project output file``() =
   p.OutputFile
   |> should equal (Some expectedOutputPath)
 
-  p.References |> should contain (normalizePath (__SOURCE_DIRECTORY__ + @"/data/DifferingOutputDir/Dir2/OutputDir2/Test1.dll"))
+  p.References
+  |> List.map (fun (s: string) -> s.Replace("//", "/"))
+  |> should contain (normalizePath (__SOURCE_DIRECTORY__ + @"/data/DifferingOutputDir/Dir1/OutputDir1/Test1.dll"))
```

Comment:
Yes, this is expected. Previously if Test2.fsproj referenced Test1.fsproj then we would use Test2/bin/Debug/Test1.dll (the CopyLocal version), but ResolveReferences returns Test1/bin/Debug/Test1.dll (the originally produced version). I think it doesn't really matter which, and the new behaviour is consistent with visualfsharp.